application/models/services/SurveyPermissions.php

```diff
@@ -415,4 +454,50 @@ public function getUserIdsWithSurveyPermissions(): array
         }
         return $aUserIds;
     }
+
+    /**
+     * Returns the list of user ids that can be used in the user selector.
+     * @return int[]|null
+     */
+    private function getFilteredUserIdList()
+    {
+        if (!isset($this->filteredUserIdList)) {
+            $this->filteredUserIdList = getUserList('onlyuidarray');
+        }
+        return $this->filteredUserIdList;
```

Comment:
Maybe you want to return the empty list here instead of null? Using `??` for example.

Reply:
return https://github.com/LimeSurvey/LimeSurvey/blob/de071e01d299b41cc79cc4067f06760ef351da31/application/helpers/common_helper.php#L664 in all case.

Then array.